server/@types/express/index.d.ts

```diff
@@ -1,6 +1,7 @@
 declare module 'express-session' {
   // Declare that the session will potentially contain these additional fields
   export interface SessionData {
+    returnUrl: query
```

Comment:
I know we have used returnUrl before, but might it be better to use a name that is more representative, like `journeyStartUrl` or `restartUrl`? 

For me, `returnUrl` is used to hold the place to go when a journey/action has completed, e.g. on logon, which isn't the case here

Reply:
Yeah makes sense. Will update.